resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+restartfile=<%= EZBake::Config[:restart_file] %>
+timeout=<%= EZBake::Config[:start_timeout] %>
```

Comment:
I think it's okay to use the start timeout here and just use 1/10 of it, as we are below in the while loop. Eventually it might make sense to break out into its own ezbake setting, but i have trouble imagining an app where reload time won't be proportional to startup time.